### PR TITLE
Added: French translation

### PR DESCRIPTION
**Pull request by floreal**
Hello,
I added here a French translation file. would you integrate it please ?
I hope this is ok for you.

Regards.

### CODE REVIEW

**Comment by madroach** [comment]
Hi,

thanks a lot. Please add your name to the copyright notice at the top, adapt the description and call the package translation::fr.

Christopher

**Comment by floreal** [comment]
Hello again,
Thanks for you review. I fixed it. I'd like to quickly review some translation i'm not sure with before you merge it.

Regards,
Floréal

File: translations/fr.pm [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  "Envoyez un message à [_1] avec pour sujet « help » (sans guillemets) pour connaître les listes de diffusion disponibles..",
+
+  'Description of list [_1]:' =>
+  'Desciption de la liste [_1]:',
+
+  'No description available.' =>
+  'Pas de description disponible.',
+
+  '[*,_1,user is,users are,No user is] subscribed to [_2]:' =>
+  "[*,_1,utilisateur est,utilisateur sont,Pesonne n'est] inscrit à [_2]:",
```
Comment (floreal) [inline translations/fr.pm]: utilisateur sont => utilisateur**s** sont

**Comment by floreal** [comment]
Looks like in need to make a couple on fixes, actually :)

File: translations/fr.pm [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  'There is no authentication request with such code ([_1]) or the requst is invalid.' =>
+  "Il n'y a pas de requête d'authentification avec ce code ([_1]), ou la requête est invalide.",
+
+  'Resend your request to Minimalist.' =>
+  'Veuillez renvoyer votre requête à Minimalist.',
+
+  'You are not allowed to get subscriptions of other users.' =>
+  "Vous n'avez pas l'autorisation d'accéder aux inscriptions d'autres utilisateurs.",
+
+  'Current subscriptions of [_1]:' =>
```
Comment (floreal) [inline translations/fr.pm]: According to code it refers to an ``$email`` variable which appears to be an user, so my translation fits properly

**Pull request merged**